core/whistle-1.0.0/src/wh_util.erl

```diff
@@ -632,6 +633,12 @@ spawn(Module, Function, Arguments) ->
                          _ = put_callid(CallId),
                          erlang:apply(Module, Function, Arguments)
                  end).
+spawn(Fun, Arguments) ->
+    CallId = get_callid(),
+    erlang:spawn(fun() ->
+                         _ = put_callid(CallId),
+                         erlang:apply(Fun, Arguments)
+                 end).
```

Comment:
I really like this!

Remove `spawn/3` and replace each of its calls with `spawn/2` and we'll have full coverage with xref!